app/Services/Email.php

```diff
@@ -137,6 +139,16 @@ protected function setupEmail()
 
             $tokens = $this->defineTokens($user);
             $this->allMessages[$key]['message'] = $this->processMessage($tokens, $this->message);
+
+            // Leaderboard messages get an extra leaderboard variable to be sent to the email template.
+            // @TODO - move into smaller function that gets the leaderboard and then add it to the allMessages array here.
+            // if ($this->message->type == 'leaderboard') {
+            //     $list = $this->manager->catalogUsers($this->users);
+
+            //     $this->allMessages[$key]['message']['leaderboard'] = $list['active'];
+            // }
+
+            // @TODO - create a smaller function that can be called here that gets the top 3 reportbacks.
```

Comment:
Comments for future reference right? For the upcoming leaderboard addition to messages?